backend/api/urls.py

```diff
@@ -3,4 +3,6 @@
 
 urlpatterns = [
     path("", view=views.index, name="index"),
+    path("quizzes/", view=views.quiz_list, name="quizzes-list"),
```

Comment:
I suggest we rename `quiz_list` to `list_quizzes`. Same for `question_list`.

Also, your names don't seem consistent. Use of `-` and `_`.

Reply:
I think it's some kind of convention, using `-` in urls `name`, instead of the usual `_`

Reply:
Yeah I know but what I'm saying is that the name for quiz is using dashes while the name for question is not

Reply:
Ah okay, didn't see it. Thanks